common/djangoapps/third_party_auth/models.py

```diff
@@ -379,6 +392,9 @@ class SAMLProviderConfig(ProviderConfig):
     attr_email = models.CharField(
         max_length=128, blank=True, verbose_name="Email Attribute",
         help_text="URN of SAML attribute containing the user's email address[es]. Leave blank for default.")
+    attr_country = models.CharField(
+        max_length=128, blank=True, verbose_name="Email Attribute",
+        help_text="URN of SAML attribute containing the user's ISO-3166-1 Alpha-2 country code. Leave blank for default.")
```

Comment:
This says "Leave blank for default.", but [unlike the other attributes](https://github.com/python-social-auth/social-core/blob/ace9733/social_core/backends/saml.py#L55-L66), I don't think that any default is implemented.

If there were a default, I think we would want it to be `urn:oid:2.5.4.6` - you could put that as an example in the help string - `e.g. "urn:oid:2.5.4.6" or leave blank to not expect a country attribute.`